parsl/tests/test_bash_apps/test_stdout.py

```diff
@@ -34,6 +34,7 @@ def echo_to_streams(msg, stderr=None, stdout=None):
 ]
 
 
+@pytest.mark.shared_fs
```

Comment:
this one never gets as far as needing to see any file that was created - all the situations are meant to break